marc_to_solr/spec/lib/config_spec.rb

```diff
@@ -43,7 +43,43 @@ def fixture_record(fixture_name)
 
   describe "alma loading" do
     it "can map an alma record" do
-      record = @indexer.map_record(fixture_record('alma1'))
+      record = @indexer.map_record(fixture_record('99211662100521'))
+    end
+  end
+  describe 'the cataloged_date from publishing job' do
+    describe "the date cataloged facets" do
+      context "When the record has 876d and 951w fields" do
+        it "will index the 876d field" do
+          expect(fixture_record('99211662100521')['951']['w']).to be_truthy
+          expect(fixture_record('99211662100521')['876']['d']).to be_truthy
+          expect(fixture_record('99211662100521')['950']['b']).to be_truthy
+          record = @indexer.map_record(fixture_record('99211662100521'))
+          expect(Time.parse(record['cataloged_tdt'].first).utc.strftime("%Y-%m-%dT%H:%M:%SZ")).to be_truthy
```

Comment:
I'm not sure we're testing that the desired value is the one we indexed. Can we use the specific value from the fixture?

Reply:
Yes I will change this. I had some circleci failures because of the timezone this is whay I tested to be present but will look into it more.